hack/do-janitor/do-janitor.go

```diff
@@ -104,6 +105,29 @@ func main() {
 		}
 	}
 
+	keys, err := keyList(ctx, client)
+	if err != nil {
+		log.Fatalf("failed to list keys: %+v", err.Error())
+	}
+
+	for _, key := range keys {
+		// we only care to delete keys that start with "capdo"
+		match, err := regexp.MatchString(`^capdo`, key.Name)
+		if match != true {
```

Comment:
maybe `if !match`

Reply:
Yeah, that's a good suggestion. I saw that in the linter output too. But it also complained about the performance of MatchString. So I removed it altogether in favor of a simple string comparison for the beginning part of the string. Wdyt about the new change?